pl_bolts/callbacks/data_monitor.py

```diff
@@ -228,7 +229,14 @@ def __init__(self, log_every_n_steps: int = None):
         """
         super().__init__(log_every_n_steps=log_every_n_steps)
 
-    def on_train_batch_start(self, trainer, pl_module, batch, *args, **kwargs):
+    def on_train_batch_start(
+            self,
+            trainer: Trainer,
+            pl_module: LightningModule,
+            batch: Sequence,
+            *args: int,
+            **kwargs: int,
```

Comment:
Shouldn't these be the following?:
```suggestion
            *args: Any,
            **kwargs: Any,
```
It seems that mypy doesn't raise any errors with `int` at the moment, but I guess it may produce some in the future.

Reply:
Ahh you're right. I mismatch from the `super().on_train_batch_start()`